napi-inl.h

```diff
@@ -2031,8 +2044,20 @@ inline const std::string& Error::Message() const NAPI_NOEXCEPT {
 inline void Error::ThrowAsJavaScriptException() const {
   HandleScope scope(_env);
   if (!IsEmpty()) {
+
+    // We intentionally don't use `NAPI_THROW_*` macros here to point
```

Comment:
I don't understand "to point that there is no possible recursion". Perhaps, "to *ensure* that there is no possible recursion"?

Reply:
agree